src/main/java/com/slimebot/util/Util.java

```diff
@@ -42,4 +47,51 @@ public static boolean isValidURL(String url) {
 			return false;
 		}
 	}
+
+	public static Instant timestamp() {
+		return new Date().toInstant().atZone(ZoneId.systemDefault()).toInstant();
+	}
+
+	public static String padRight(String s, int n) {
+		return padRight(s, ' ', n);
+	}
+
+	public static String padLeft(String s, int n) {
+		return padLeft(s, ' ', n);
+	}
+
+	public static String padRight(String s, char padChar, int n) {
+		StringBuilder sb = new StringBuilder(s);
+		while (sb.length()<n) sb.append(padChar);
+		return sb.toString();
```

Comment:
return s + padString,repeat(n);

Reply:
no. you have not understood the meaning behind pad. not a certain number of characters should be added, the specified character should be appended until the string has reached a certain length. so if String s is longer, less characters will be added.

Reply:
Still:
```java
public static String padRight(String s, chat padChar, int n) {
  return s + String.valueOf(padChar).repeat(n - s.length());
}
```